interfaces/apparmor/template.go

```diff
@@ -17,10 +17,15 @@
  *
  */
 
-package interfaces
+package apparmor
+
+// defaultTemplate contains default apparmor template.
+//
+// It can be overridden for testing using MockTemplate().
+var defaultTemplate = realDefaultTemplate
 
 // http://bazaar.launchpad.net/~ubuntu-security/ubuntu-core-security/trunk/view/head:/data/apparmor/templates/ubuntu-core/16.04/default
-const defaultAppArmorTemplate = `
+const realDefaultTemplate = `
```

Comment:
I wanted to keep the real template, that cannot be mocked, in a constant for reference.